pkg/operation/purge/purge.go

```diff
@@ -126,17 +132,49 @@ func (p *Purge) GarbageCollect() error {
 		return err
 	}
 	glog.V(2).Infof("Kube-apiserver returns %d csr", len(csrList.Items))
+	purged := 0
 	for _, elt := range csrList.Items {
 		glog.V(4).Infof("Got csr/%s", elt.Name)
-		for _, fn := range p.conf.ShouldPurge {
+		for _, fn := range p.conf.ShouldGC {
 			if !fn(&elt, p.conf.GracePeriod) {
 				continue
 			}
 			err := p.Delete(elt.Name)
 			if err != nil {
 				return err
 			}
+			purged++
 		}
 	}
+	if purged > 0 {
+		glog.V(0).Infof("Successfully garbage collected %d csr", purged)
+	}
 	return nil
 }
+
+// GarbageCollectorLoop runs the GC on ticker, returns on error or SIGINT/TERM
+func (p *Purge) GarbageCollectorLoop() error {
```

Comment:
You have `newGarbageCollect`, `GarbageCollect` and `GarbageCollectorLoop`, to be more consistent, I would change to `newGarbageCollector`, `GarbageCollect` and `GarbageCollectLoop`